### ISSUE TITLE
artisan migrate failure for installation instructions for arch linux

### ISSUE THREAD

**Issue by user171** [pixelfed/support#80]
I'm trying to follow exactly the instructions from https://docs.pixelfed.org/installing-pixelfed/guides/arch.html#pixelfed-setup

Everything seems to work until the end when I run the `php artisan migrate --force` command, which returns:

`Access denied for user 'pixelfed'@'localhost' (using password: YES) (SQL: select * from information_schema.tables where table_schema = pixelfed and table_name = migrations and table_type = 'BASE TABLE')`

I run this while being logged as the pixelfed user as per the docs. 
While being logged in as the pixelfed user, if I run `mysql -u pixelfed -pmy_password` I can login to mysql just fine.

It is not in the docs but I guess the mysql password must be entered in the `.env` file, in the `DB_PASSWORD` field. Which I did. 
I even tried with an empty password. 

Any idea what I'm doing wrong?

**Comment by trwnh** [comment]
```
php artisan config:cache
php artisan route:cache
php artisan migrate --force
```
already there under step 5, deploy :)

**Comment by user171** [comment]
Thanks for your message. 
Yes indeed I checked the docs after posting my comment and realized it was there. So I went checking my command history and saw that I had executed the command. But yet I had the issue anyway.

So, what I believe happened is that I followed the guide for arch line by line, and at the step `Setup environment variables and nginx` I did copy the `.env` file and edited what is in the docs. But at that time I should also have set the password for mysql, which I did not. 
Then when I first encountered the mysql error I found out that the password should be set in the `.env`, which I updated. 
But then it did not work until I executed  `php artisan config:cache` again.

So, maybe I should have looked closer to the `.env` file the first time I edited it, but perhaps it's a good idea to add the password editing step in the docs.

And also, if I understand correctly this command is necessary to take into account the changes in the `.env` file. Am I right?
If true then maybe it's a good thing to state in the docs.

I can do a PR if you find it helpful. 

cheers
